torch/distributions/transforms.py

```diff
@@ -146,30 +146,34 @@ class _InverseTransform(Transform):
     """
     def __init__(self, transform):
         super(_InverseTransform, self).__init__()
-        self.inv = transform
+        self._inv = transform
 
     @constraints.dependent_property
     def domain(self):
-        return self.inv.codomain
+        return self._inv.codomain
 
     @constraints.dependent_property
     def codomain(self):
-        return self.inv.domain
+        return self._inv.domain
 
     @property
     def bijective(self):
-        return self.inv.bijective
+        return self._inv.bijective
+
+    @property
+    def inv(self):
+        return self._inv
```

Comment:
I think it could have stayed as `self.inv`

Reply:
That gave me a "cannot set attribute" error somewhere. I only renamed it to get tests to pass.

Reply:
```
    def __init__(self, transform):
        super(_InverseTransform, self).__init__()
>       self.inv = transform
E       AttributeError: can't set attribute
```